packages/flex-plugin-scripts/src/config/index.ts

```diff
@@ -6,20 +7,27 @@ import { Configuration as WebpackDevConfigurations } from 'webpack-dev-server';
 import webpackFactory from './webpack.config';
 import devFactory from './webpack.dev';
 
-export type Environment = 'production' | 'development';
 type ConfigurationTypes = 'webpack' | 'devServer';
 interface Configurations {
   webpack: WebpackConfigurations;
   devServer: WebpackDevConfigurations;
 }
 
+/**
+ * Returns the configuration; if customer has provided a webpack.config.js, then the generated config is passed to their Function for modification
+ * @param name  the configuration name
+ * @param env   the environment
+ */
 const getConfiguration = <C extends ConfigurationTypes>(name: ConfigurationTypes, env: Environment): Configurations[C] => {
   const pwd = process.cwd();
+  const isProd = env === 'production';
+  const isDev = env === 'development';
+
   if (name === 'webpack') {
     const path = join(pwd, 'webpack.config.js');
     const config = webpackFactory(env);
     if (checkFilesExist(path)) {
-      return require(path)(config);
+      return require(path)(config, { isProd, isDev });
```

Comment:
why are both needed? can they ever both be true or both be false? 🤔

Reply:
We'll also have a `isTest` later